aip/general/0193.md

```diff
@@ -34,272 +34,154 @@ is necessary, you **should** provide a link where a reader can get more
 information or ask questions to help resolve the issue. It is also
 important to [set the right tone][writing-tone] when writing messages.
 
-### Error Response
+The following sections describe the fields of `google.rpc.Status`.
 
-#### JSON representation
+### Status.message
 
-A JSON representation of an error response might look like the following
-example.
+The `message` field is a a developer-facing, human-readable "debug message"
+which **should** be in English. (Localized messages are expressed using
+a `LocalizedMessage` within the `details` field.) Any dynamic aspects of
+the message **must** be included as metadata within the `ErrorInfo` message
```

Comment:
I agree with your position 100%. I think I was getting confused with semantic meaning vs. concrete reference reading the document top to bottom.

Reply:
That said, my personal tendency is to explicitly state "`message` value" or "the value of `message`" or "the content of `message`" rather than depend on context clues (like code formatting) to do that "comprehension lifting". I'm not too fussed about it though.

Reply:
Agreed. I'd only put the word "message" in quotes when you're referring directly to the field.